video/mp_image.h

```diff
@@ -38,6 +38,10 @@
 #define MP_IMGFIELD_REPEAT_FIRST 0x04
 #define MP_IMGFIELD_INTERLACED 0x20
 
+#define MP_FIELD_PARITY_AUTO -1
+#define MP_FIELD_PARITY_TFF 0
+#define MP_FIELD_PARITY_BFF 1
```

Comment:
Sorry to have to do another round, but now that we see the existing defines are prefixed with `MP_IMGFIELD`, the new ones should be as well.

Reply:
I've changed the name, although I'm a bit conflicted now that I'm looking at it. The previous defines prefixed with `MP_IMGFIELD` are flags for the `fields` member of `mp_image` while these would be used for filters. I think I would rather revert it back to just `MP_FIELD` or maybe 'MP_FILTER_FIELD'?

Reply:
You're right. And in fact, it's clear that `option.c` can include from anywhere, really, so you should go back to how it originally was. Sorry for the churn!

Reply:
Alright, moved them back to `refqueue.h` and included the header